cmd/ak/cmd/vars/set.go

```diff
@@ -32,16 +32,35 @@ var setCmd = common.StandardCommand(&cobra.Command{
 		}
 
 		var value string
-		if len(args) == 2 {
-			value = args[1]
+
+		isValueSet := len(args) > 1
+
+		if isValueSet {
+			if args[1] != "-" {
+				value = args[1]
+			} else {
+				const maxVarSize = 1 << 20 // 1MB
+				r := io.LimitReader(os.Stdin, maxVarSize)
+				data, err := io.ReadAll(r)
+				if err != nil {
+					return err
+				}
+				value = string(data)
+			}
 		} else {
```

Comment:
Itay, for me it feels not the right solution to fetch the variable from the DB in order not to reset it if it wasn't provided. It's a lot of logic/checks which could and IMO should be avoided
- let's first discuss whether we allow empty values (I'm still in favor of not allowing them)
- if we are allowing empty values, then we should treat them as normal values, thus allowing user to override any value with empty one. For this user should always(!) provide two(!) args, if the second arg is explicitly "", then value will be overrided with the empty one